sklearn/manifold/_t_sne.py

```diff
@@ -990,6 +990,8 @@ def _fit(self, X, skip_num_points=0):
                 svd_solver="randomized",
                 random_state=random_state,
             )
+            # Alays output a numpy array, no matter what is configured globally
+            pca.set_output(transform="default")
```

Comment:
Is there a way to make work t-SNE with both arrays and dataframe instead of locally setting the configuration of the estimator, similar to https://github.com/scikit-learn/scikit-learn/pull/24932?

Reply:
Note it as resolved on my side.